visualize-admin/visualization-tool
# fix: Use components with hierarchies when retrieving cached data in configurator state

Fixes #1148.

This PR uses correct query when retrieving cached data in configurator state. This fixes initial area layer filtering, as we now pass components with hierarchies to the function that initializes a chart.

### How to test
1. Go to [this link](https://visualization-tool-git-fix-initial-map-filters-ixt1.vercel.app/en/create/new?cube=https://environment.ld.admin.ch/foen/nfi/nfi_C-20/cube/2023-2).
2. Change chart type to map.
3. See that top-level regions have been selected by default (Jura, Plateau, ...).
